src/components/inputs/SliderInput.tsx

```diff
@@ -21,7 +21,7 @@ interface SliderInputProps extends InputProps {
     max: number;
     def?: number;
     accentColor: string;
-    hideEnds?: boolean;
+    ends?: Array<string | number> | null;
```

Comment:
Use a tuple type.

Tuples aren't a runtime type/class in JS (yet), but TS has a tuple type.

```suggestion
    ends?: [string | number, string | number] | null;
```

Reply:
Definitely need to wait to merge stuff until you've taken a look haha